client/src/pages/onboarding/availability/index.js

```diff
@@ -71,18 +62,26 @@ function AvailabilityPage(props) {
   function onChangeAvailableDays(e) {
     console.log(e.target.value);
     setUsers({
-      name: users.name,
-      email: users.email,
-      avatarUrl: users.avatarUrl,
       timeZone: users.timeZone,
       availableHoursFrom: users.availableHoursFrom,
       availableHoursTo: users.availableHoursTo,
-      availableDays: users.availableDays.push(e.target.value),
+      availableDays: addDays(e.target.value),
       calendarUrl: users.calendarUrl,
     });
     console.log("Users", users);
   }
 
+  function addDays(value) {
```

Comment:
Here's a suggestion since "available days" are finite we could have a default object that looks like:

```
{
  "Monday": true,
  "Tuesday": true,
   ...
  "Sunday": true
}
```

And if they uncheck one of those days, we turn that object day to false

to store it in the db, we can just push all the true values' keys into an array

while i'm not opposed this way of doing it because it's not an issue with speed or efficiency but just a suggestion and insight as to how i would have done it :)

Reply:
Yeah I was thinking something like this my idea was [0,1,1,1,1,1,0] for Mon-Friday. Thanks Bonnie.